scikeras/wrappers.py

```diff
@@ -463,6 +464,8 @@ def _fit_keras_model(
             Number of epochs for which the model will be trained.
         initial_epoch : int
             Epoch at which to begin training.
+        **kwargs : Dict[str, Any]
+            Extra arguments to route to ``Model.fit``.
```

Comment:
What arguments can these be? I would include a link to the Keras API docs here. https://keras.io/api/models/model_training_apis/

Reply:
The current docs are at https://www.tensorflow.org/api_docs/python/tf/keras/Model?version=nightly#fit but yeah, I'll include that.